setup.py

```diff
@@ -83,7 +83,7 @@ def run_tests(self):
     keywords="api graphql protocol rest relay graphene",
     packages=find_packages(exclude=["examples*"]),
     install_requires=[
-        "graphql-core>=3.1,<3.3",
+        "graphql-core>=3.1,<3.4",
```

Comment:
There's one issue with`graphql-relay>=3.1,<3.3` on line 87. Currently the latest graphql-rely version is 3.2 which does not support graphql-core 3.3 or 3.3.* versions. So we still get dependency conflicts.

@etripier do we need to use a forked version here for graphq-relay as well?